tests/test_routes.py

```diff
@@ -543,3 +543,32 @@ def test_update_item_item_not_found(self):
             content_type="application/json",
         )
         self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+
+# ----------------------------------------------------------
```

Comment:
Why are the tests in routes?

Reply:
Since both model test cases and endpoint test cases are needed, "test_routes.py" contains the endpoint test cases here.